src/pages/en/guides/imports.md

```diff
@@ -210,3 +210,39 @@ const data = JSON.parse(json);
 
 <span>Version: {data.version}</span>
 ```
+
+## Extending file type support
+
+You can import file types which aren't natively supported by Astro. There are many **Vite** and compatible **Rollup** plugins you can use. Learn where to find the plugins you need in the [Finding Plugins](https://vitejs.dev/guide/using-plugins.html#finding-plugins) section of Vite Documentation.
+
+For example, the YAML data format isn't natively supported by Astro, but you can add support for it by using a compatible Rollup plugin:
+
+1. Install `@rollup/plugin-yaml`:
+
+```bash
+npm install @rollup/plugin-yaml --save-dev
+```
+
+2. Import the plugin in your `astro.config.mjs` and add it to the plugins array:
+
+```js title="astro.config.mjs" ins={2,6,7,8}
+import { defineConfig } from 'astro/config';
+import yaml from '@rollup/plugin-yaml';
+
+export default defineConfig(
+{
+  vite: {
+    plugins: [yaml()]
+  }
+});
+```
+
+:::note
+Refer to your plugin's documentation to learn what configuration options it has, and how to correctly install it.
+:::
```

Comment:
Same indentation note here. And what do you think about a custom heading like this for the note? I think the advice is good but because it’s inside the YAML plugin instructions, it might be good to add a title for a little more context.

````suggestion
    :::note[Using a different plugin?]
    Refer to your plugin's documentation to learn what configuration options it has, and how to correctly install it.
    :::
````